Trident/offsetfinder.c

```diff
@@ -598,6 +608,7 @@ uint32_t find_vm_kernel_addrperm(void) {
 
 uint32_t find_kernel_pmap(void) {
     switch (target_environment) {
+        case iPhone41_iOS920: return 0x3f8444;
```

Comment:
Ph the new line should be 910

I forgot to change it when adding the offset sorry. 

Can you please change that?